nf_core/pipeline-template/{{cookiecutter.name_noslash}}/main.nf

```diff
@@ -95,23 +89,23 @@ if( workflow.profile == 'awsbatch') {
 }
 
 // Stage config files
-ch_multiqc_config = Channel.fromPath(params.multiqc_config)
-ch_output_docs = Channel.fromPath("$baseDir/docs/output.md")
+ch_multiqc_config = file(params.multiqc_config, checkIfExists: true)
+ch_output_docs = file("$baseDir/docs/output.md", checkIfExists: true)
 
 /*
  * Create a channel for input read files
  */
-if(params.readPaths){
-    if(params.singleEnd){
+if (params.readPaths) {
```

Comment:
This whole section needs refactoring anyway... https://github.com/nf-core/tools/issues/211

Reply:
Will we need to bump the minimum version of Nextflow too because the functionality came out in more recent releases?
https://github.com/nf-core/tools/issues/211#issuecomment-480798735

Reply:
Yup :)

Reply:
Shall we put this on hold until `1.8` then in the name of time?

Reply:
Maybe too short notice for the 1.7 release? Rather push some of these things to 1.8 and make that release also sooner than later?